providers/base/units/ethernet/manifest.pxu

```diff
@@ -2,3 +2,8 @@ unit: manifest entry
 id: has_ethernet_adapter
 _name: An Ethernet Port
 value-type: bool
+
+unit: manifest entry
+id: has_ethernet_wake_on_lan_support
+_name: Wake-On-Lan support by Ethernet Port
```

Comment:
Using [Wikipedia page](https://en.wikipedia.org/wiki/Wake-on-LAN) as reference on naming the "Wake-on-LAN" feature

```suggestion
_name: Wake-on-LAN support through Ethernet port
```